src/detection/PsfFormatter.cc

```diff
@@ -22,14 +22,13 @@
 #include "lsst/daf/persistence/BoostStorage.h"
 #include "lsst/daf/persistence/XmlStorage.h"
 #include "lsst/pex/exceptions.h"
-#include "lsst/pex/logging/Trace.h"
+#include "lsst/log/Log.h"
 #include "lsst/pex/policy/Policy.h"
 
 BOOST_CLASS_EXPORT(lsst::afw::detection::Psf)
 
-#define EXEC_TRACE  20
-static void execTrace(std::string s, int level = EXEC_TRACE) {
-    lsst::pex::logging::Trace("afw.detection.PsfFormatter", level, s);
+namespace {
+LOG_LOGGER _log = LOG_GET("afw.detection.PsfFormatter");
```

Comment:
Is it always safe to construct a logger at static initialization time?  I'm a bit worried about [static initialization order](https://isocpp.org/wiki/faq/ctors#static-init-order) here.